NuGet.config

```diff
@@ -10,8 +10,6 @@
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="myget-roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
-    <add key="myget-roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
```

Comment:
Just removing these isn't working. Adding https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json to these sources should get everything you need.